src/Data/Avro/Encoding/ToAvro.hs

```diff
@@ -127,6 +127,10 @@ instance ToAvro Float where
   toAvro s _        = error ("Unable to encode Float as: " <> show s)
   {-# INLINE toAvro #-}
 
+instance ToAvro () where
+  toAvro S.Null () = putI 0
```

Comment:
Hmm... Doesn't it violate specification? https://avro.apache.org/docs/current/spec.html#Encodings

> null is written as zero bytes.

Reply:
you're absolutely right, thanks for pointing it out

Reply:
I've amended the commit with this fix